meteor/bin/test.sh

```diff
@@ -57,14 +57,14 @@ export TEST_FILE="$REPO_HOME/meteor/tests/test-package.html"
 
 cd "$REPO_HOME/meteor/test-package"
 MOCHA_RUN_ORDER='parallel'
-spacejam test-packages --phantomjs-script "$REPO_HOME/meteor/tests/test-app-tests.js" --driver-package practicalmeteor:mocha ./
+spacejam test-packages --phantomjs-script "$REPO_HOME/meteor/tests/phantomjs-test-script.js" --driver-package practicalmeteor:mocha ./
 createCompareFile "$?" "$EXIT_STATUS" "test-package-parallel"
 EXIT_STATUS="$?"
 
 
 cd "$REPO_HOME/meteor/test-package"
 MOCHA_RUN_ORDER='serial'
-spacejam test-packages --phantomjs-script "$REPO_HOME/meteor/tests/test-app-tests.js" --driver-package practicalmeteor:mocha ./
+spacejam test-packages --phantomjs-script "$REPO_HOME/meteor/tests/phantomjs-test-script.js" --driver-package practicalmeteor:mocha ./
```

Comment:
This is our mocha self test script. Let's call it someday meteor-mocha-self-test.js